src/gatsby/models/authors/parse-link.js

```diff
@@ -0,0 +1,94 @@
+/*
+   Processor builder: makes a function that turns a string into processed Link metadata
+
+   Takes "Mod functions" as parameters, returns a function that starts with {} and pipes
+   that through each function, then returns the result.
+
+   signature: ({ scheme: string, host: string, pathname: string}, input)
+   example:      ^ https://      ^ mysite.com  ^ /rest/of/the/url ^ an object
+
+   The second parameter will be the current state of the input, and usually will
+   have to be spread into your return result.
+ */
+function processor(...mods) {
+  return function process(groups) {
+    return mods.reduce(
+      (currentResult, _, i) => mods[i](groups, currentResult),
+      {}
+    )
+  }
+}
+
+// Processor helpers
+
+function trimSlashes(input) {
+  return /^\/?(.*?)\/?$/.exec(input)[1]
```

Comment:
same here - are you sure it actually trims the trailing slash? do we care about it at all btw?

Reply:
I know it works on both because I've tested it alone, but only the leading slash matters because it's how we extract the username from the path on many sites (`website.com/username`)

Reply:
yep, missed the `*?` - lazy part of it